src/pages/Login/Login.js

```diff
@@ -1,8 +1,100 @@
-import React, { Component } from 'react';
+import React from 'react';
+import './Login.scss';
+
+class Login extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      userId: '',
+      userPw: '',
+    };
+  }
+
+  handleChange = e => {
+    const { name, value } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  // 로그인 성공 여부
+  loginFetch = () => {
+    fetch('http://10.58.5.217:8000/users/signin', {
+      method: 'POST',
+      body: JSON.stringify({
+        email: this.state.userId,
+        password: this.state.userPw,
+      }),
+    })
+      .then(response => response.json())
+      .then(result => {
+        if (result.token !== undefined) {
```

Comment:
undefined는 falsy값이니까 굳이 !== 로 비교해줄 필요 없어보입니다!
```suggestion
        if (result.token) {
```

Reply:
수정 완료하였습니다!! 😎